src/index.html

```diff
@@ -128,6 +128,7 @@ <h3 class="text--headline text--headline--3 space--mt--large space--mb">The Don
           <li><strong>Do not</strong> behave as though you know best</li>
           <li><strong>Do not</strong> take credit for the labor of those who are marginalized and did the work before you stepped into the picture</li>
           <li><strong>Do not</strong> assume that every member of an underinvested group feels oppressed</li>
+          <li><strong>Do not</strong> focus on the "what" (riots and looting), focus on the "why" (racism)</li>
```

Comment:
Is there a way to re-write this to be more general? The language of this guide is meant to be open enough to speak to -isms beyond racism.

Reply:
This is great, thanks so much!